pages/installation/manual_setup.rst

```diff
@@ -97,46 +97,42 @@ log files to grow without bounds using the `RollingFileAppender <https://logging
 One such example ``log4j2.xml`` configuration is shown below::
 
   <?xml version="1.0" encoding="UTF-8"?>
-  <Configuration packages="org.graylog2.log4j" shutdownHook="disable">
+<Configuration packages="org.graylog2.log4j" shutdownHook="disable">
```

Comment:
The indentation is broken:

![image](https://user-images.githubusercontent.com/461/84668538-0f623800-af24-11ea-8327-cb229fec08d6.png)